Fix issues/483 and add a basic Node example

nicolaslevy opened the pull request:
### Description

* Fixes issues described in https://github.com/dailymotion/vast-client-js/issues/483 
* Add a Node example 
 
### Issue

https://github.com/dailymotion/vast-client-js/issues/483 

### Type
- [ ] Breaking change
- [ ] Enhancement
- [x] Fix
- [ ] Documentation
- [x] Tooling

ZacharieTFR commented on dist/chunks/xmldom-623c1b6b.min.js:
Please do not push the dist, we will do it during the next release 😉

nicolaslevy replied:
:sweat_smile: done in https://github.com/dailymotion/vast-client-js/pull/484/commits/f910ea0e3a9acf69aac74f43cf4b8528bc4fa7b1

nicolaslevy replied:
oh... my bad, I deleted them :P Let me restore the original dist folder

nicolaslevy replied:
Yeap, now the dist folder has no changes. Solved in commit https://github.com/dailymotion/vast-client-js/pull/484/commits/2b48c61d6eac59164e400ef689255c64a2f11f6b

ZacharieTFR commented on examples/node/package.json:
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "test-node",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "start": "node index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "description": ""
```
Lets re-use the same information than the root package
```suggestion
  "name": "VASTClient-node-example",
  "version": "1.0.0",
  "main": "index.js",
  "type": "module",
  "scripts": {
    "start": "node index.js",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "Dailymotion <vast-client@dailymotion.com> (https://developer.dailymotion.com/)"
  "license": "MIT",
  "description": "VAST Client node usage example"
```

ZacharieTFR reviewed (changes requested):
Hello, thank you for your contribution, it is much appreciated! I've left a few comments

nicolaslevy commented:
Thank you @ZacharieTFR, I made those changes

ZacharieTFR commented on examples/node/index.js:
```diff
@@ -0,0 +1,11 @@
+    console.log(parsedVast);
+}
+
+await parseSimpleInline();
```
The EOF is missing 
```suggestion
await parseSimpleInline();

```

ZacharieTFR reviewed (commented):
As you moved the example folder and added the node, can you update the readme aswell ? 
https://github.com/dailymotion/vast-client-js/blob/d6420de113cfd8180cc943e0079e9f2c20374436/README.md?plain=1#L18

nicolaslevy commented:
> As you moved the example folder and added the node, can you update the readme aswell ?
> 
> https://github.com/dailymotion/vast-client-js/blob/d6420de113cfd8180cc943e0079e9f2c20374436/README.md?plain=1#L18

I changed this text in the last commit:
> To explore a practical example of how the VAST client can be implemented in web applications please visit this [link](https://githubbox.com/dailymotion/vast-client-js/tree/master/examples/web). Additionally, use this [link](https://githubbox.com/dailymotion/vast-client-js/tree/master/examples/node) to learn how to use the VAST client in a basic Node application.

@ZacharieTFR  Let me know what you think.

ZacharieTFR reviewed (approved):
LGTM !

nicolaslevy commented:
Awesome!! just FYI: we are using this library in this project https://github.com/montevideo-tech/vast-2-sgai/ .A  basic VAST-2-SGAI (VAST to Asset List and ListMPD) opens-ource microservice :)
